lib/chef/resource/windows_feature_powershell.rb

```diff
@@ -215,7 +215,7 @@ def reload_cached_powershell_data
         # fetch the list of available feature names and state in JSON and parse the JSON
         def parsed_feature_list
           # Grab raw feature information from WindowsFeature
-          raw_list_of_features = powershell_out!("Get-WindowsFeature | Select-Object -Property Name,InstallState | ConvertTo-Json -Compress", timeout: new_resource.timeout).stdout
+          raw_list_of_features = powershell_exec!("Get-WindowsFeature | Select-Object -Property Name,InstallState | ConvertTo-Json -Compress", timeout: new_resource.timeout).result
```

Comment:
If we're using powershell_exec we don't need to convert to json. The data should be returned in JSON to start with.

Reply:
done!